test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -2008,6 +2024,101 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				Skip("No windows agent was provisioned for this Cluster Definition")
 			}
 		})
+		It("should be able to deploy containers after restart", func() {
```

Comment:
nit: reword to something like 'should expect containers to be recreated after restart'